etc/logrotate.d/tendrl-node-agent_logrotate.conf

```diff
@@ -6,5 +6,5 @@
     missingok
     compress
     notifempty
-    create 0664 root root
+    copytruncate
```

Comment:
instead of creating a new file,  it will keep the old file and continue logging. So create has no effect here. https://linux.die.net/man/8/logrotate